databases/week1/query1.sql

```diff
@@ -0,0 +1,25 @@
+SELECT * FROM task;
+
+
+
+--1.Find out how many tasks are in the task table
+select count(*) as total_task from task;
+--2.Find out how many tasks in the task table do not have a valid due date
+select * from task where due_date IS NULL;
+--3.Find all the tasks that are marked as done
+SELECT title FROM task WHERE status_id = 3;
```

Comment:
this is correct only if you knew the status id beforehand, you can also query by status name